flytekit/taskplugins/sidecar/task.py

```diff
@@ -0,0 +1,111 @@
+from typing import Any, Callable, Dict, Tuple, Union
+
+from flyteidl.core import tasks_pb2 as _core_task
+from google.protobuf.json_format import MessageToDict
+
+from flytekit.annotated.context_manager import FlyteContext, RegistrationSettings
+from flytekit.annotated.promise import Promise
+from flytekit.annotated.python_function_task import PythonFunctionTask
+from flytekit.annotated.task import TaskPlugins
+from flytekit.common.exceptions import user as _user_exceptions
+from flytekit.models import task as _task_model
+from flytekit.models import task as _task_models
+from flytekit.plugins import k8s as _lazy_k8s
```

Comment:
I do not think you need to do this. But, I guess lets keep it. I need to move all the _lazy_k8s logic to the plugins